react/Autosuggest/Autosuggest.js

```diff
@@ -81,14 +81,16 @@ export default class Autosuggest extends Component {
       }
     };
 
+    const suggestionsContainer = typeof document !== 'undefined' && document.getElementById(containerProps.id);
```

Comment:
Not sure i'm keen on having every consuming component needing to implement this. Feel like we should make a higher order component that implements scroll lock and makes `ref` a required property. Then we can resolve the dom node from the ref internally to the HOC. Thoughts?